### PR TITLE
[15.0][MIG]  l10n nl xaf auditfile export

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

**Comment by astirpe** [comment]
@NL66278 do you mind to cherry pick this fix: https://github.com/OCA/l10n-netherlands/pull/371/commits/fc97a764d2240986e1572acae5b1d360c91ad58a ?

**Comment by NL66278** [comment]
@astirpe Done!

**Comment by NL66278** [comment]
@astirpe Do you have any pointer or idea why the tests in my PR are failing? I get some message on invalid taxes for my country that I cannot explain.

**Comment by astirpe** [comment]
@NL66278 it's the first time I see that error :see_no_evil: If I can find some time in the next days I will try to reproduce it and debug

File: l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            self.env["account.move"].with_context(default_move_type="out_invoice")
+        )
+        move_form.invoice_date = fields.Date().today()
+        move_form.partner_id = self.env["res.partner"].create({"name": "Partner Test"})
+        with move_form.invoice_line_ids.new() as line_form:
+            line_form.product_id = self.env["product.product"].create(
+                {"name": "product test", "standard_price": 800.0}
+            )
+        self.invoice = move_form.save()
+        self.invoice.post()
```
Comment (bosd) [inline l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py]: There is a warning on this method.
DeprecationWarning: RedirectWarning method 'post()' is a deprecated alias to 'action_post()' or _post()

**Review by astirpe — APPROVED** [review]
Looks good!

**Comment by astirpe** [comment]
This is ready for merge IMO. Do you agree @bosd ?

**Review by bosd — APPROVED** [review]
LGTM 👍

**Pull request merged**